stytch/b2b/discovery/organizations/types.go

```diff
@@ -0,0 +1,197 @@
+//     delete a key, supply a null value. Custom claims made with reserved claims (`iss`, `sub`, `aud`,
+//     `exp`, `nbf`, `iat`, `jti`) will be ignored.
+//     Total custom claims size cannot exceed four kilobytes.
+//
+//   - OrganizationLogoURL: The image URL of the Organization logo.
+//
+//   - TrustedMetadata: An arbitrary JSON object for storing application-specific data or
+//     identity-provider-specific data.
+//
+//   - SsoJitProvisioning: The authentication setting that controls the JIT provisioning of Members when
+//     authenticating via SSO. The accepted values are:
+//
+//     `ALL_ALLOWED` – new Members will be automatically provisioned upon successful authentication via any
+//     of the Organization's `sso_active_connections`.
+//
+//     `RESTRICTED` – only new Members with SSO logins that comply with
+//     `sso_jit_provisioning_allowed_connections` can be provisioned upon authentication.
+//
+//     `NOT_ALLOWED` – disable JIT provisioning via SSO.
+//
+//   - EmailAllowedDomains: An array of email domains that allow invites or JIT provisioning for new
+//     Members. This list is enforced when either `email_invites` or `email_jit_provisioning` is set to
+//     `RESTRICTED`.
+//
+//     Common domains such as `gmail.com` are not allowed. See the for the full list.
+//
+//   - EmailJitProvisioning: The authentication setting that controls how a new Member can be provisioned
+//     by authenticating via Email Magic Link. The accepted values are:
+//
+//     `RESTRICTED` – only new Members with verified emails that comply with `email_allowed_domains` can be
+//     provisioned upon authentication via Email Magic Link.
+//
+//     `NOT_ALLOWED` – disable JIT provisioning via Email Magic Link.
+//
+//   - EmailInvites: The authentication setting that controls how a new Member can be invited to an
+//     organization by email. The accepted values are:
+//
+//     `ALL_ALLOWED` – any new Member can be invited to join via email.
+//
+//     `RESTRICTED` – only new Members with verified emails that comply with `email_allowed_domains` can be
+//     invited via email.
+//
+//     `NOT_ALLOWED` – disable email invites.
+//
+//   - AuthMethods: The setting that controls which authentication methods can be used by Members of an
+//     Organization. The accepted values are:
+//
+//     `ALL_ALLOWED` – the default setting which allows all authentication methods to be used.
+//
+//     `RESTRICTED` – only methods that comply with `allowed_auth_methods` can be used for authentication.
+//     This setting does not apply to Members with `is_breakglass` set to `true`.
+//
+//   - AllowedAuthMethods:
+//     An array of allowed authentication methods. This list is enforced when `auth_methods` is set to
+//     `RESTRICTED`.
+//     The list's accepted values are: `sso`, `magic_link`, and `password`.
+type CreateParams struct {
+	IntermediateSessionToken string         `json:"intermediate_session_token,omitempty"`
```

Comment:
Hmm, the IST is only used for Discovery Organizations Create. Why is it appearing here?

Reply:
I am in the wrong file 😮‍💨